api/environments/environment_handler.go

```diff
@@ -313,7 +310,15 @@ func (eh EnvironmentHandler) getOrphanedEnvironments(appName string, radixApplic
 
 // getOrphanedEnvNames returns a slice of non-unique-names of orphaned environments
 func (eh EnvironmentHandler) getOrphanedEnvNames(app *v1.RadixApplication) []string {
+	return eh.getEnvironments(app, true)
+}
+
+// getNotOrphanedEnvNames returns a slice of non-unique-names of not-orphaned environments
+func (eh EnvironmentHandler) getNotOrphanedEnvNames(app *v1.RadixApplication) []string {
+	return eh.getEnvironments(app, false)
+}
 
+func (eh EnvironmentHandler) getEnvironments(app *v1.RadixApplication, orphaned bool) []string {
```

Comment:
name "orphaned" "isOrphaned" for better readability

Reply:
Fixed